index.js

```diff
@@ -231,6 +245,9 @@ class TagInput<T> extends React.PureComponent<Props<T>, State> {
     const tags = [...this.props.value];
     tags.pop();
     this.props.onChange(tags);
+    if (this.props.scrollHorizontal) {
+      this.scrollToRight();
```

Comment:
I think it actually makes sense to always scroll to the end when a tag is deleted as a result of a backspace, so let's take out the `scrollHorizontal` condition here.